python/src/aiconfig/editor/client/src/utils/promptUtils.ts

```diff
@@ -84,6 +86,12 @@ export const PROMPT_SCHEMAS: Record<string, PromptSchema> = {
 
   // Local HuggingFace Parsers
   HuggingFaceText2ImageDiffusor: HuggingFaceText2ImageDiffusorPromptSchema,
+  TextGeneration: HuggingFaceTextGenerationTransformerPromptSchema,
+  Text2Image: HuggingFaceText2ImageDiffusorPromptSchema,
+  AutomaticSpeechRecognition: HuggingFaceAutomaticSpeechRecognitionPromptSchema,
+  // Text2Speech: HuggingFaceText2SpeechTransformerPromptSchema,
+  // HuggingFaceTextTranslationTransformer: HuggingFaceTextTranslationTransformerPromptSchema,
```

Comment:
Let's just use `HuggingFaceTextGenerationTransformerPromptSchema` for these two since both of these transformers are sub-classes of TextGeneration so I know at minimum text-generation input schema will be a sub-set of these two completion params

Reply:
Thanks. I made separate prompt schemas that just spread the text one in for now. We can revisit if we see actual differences, otherwise can later rename to something more general for use in these 3